onyo/lib/differs.py

```diff
@@ -59,7 +59,6 @@ def differ_move_assets(repo: OnyoRepo, operands: tuple) -> Generator[str, None,
 
 def differ_move_directories(repo: OnyoRepo, operands: tuple) -> Generator[str, None, None]:
     yield from diff_path_change(operands[0], operands[1])
-    # TODO: Fuse w/ differ_move_asset
```

Comment:
Out of curiosity, because you removed this and other TODOs regarding fusing functions: Does this not need to happen anymore or was it done in annother way?